bindings/dotnetcore/dotnet-core-binding/Microsoft.Azure.Devices.Gateway/Gateway.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Azure.Devices.Gateway
+{
+    /// <summary>
+    ///  Wrapper for Azure IoT Gateway SDK. 
+    /// </summary>
+    public class Gateway
+    {
+        [DllImport(
+            "gateway",
+            CharSet = CharSet.Ansi,
+            EntryPoint = "Gateway_CreateFromJson",
+            CallingConvention = CallingConvention.Cdecl
+        )]
+        private static extern IntPtr CreateFromJsonInternal(string file_path);
+
+
+        /// <summary>
```

Comment:
I wonder if we should simply refer to the C API documentation, and avoid this duplication.
```
/// Calls Gateway_CreateFromJson(...) in the C API
```